fdbclient/include/fdbclient/SpecialKeySpace.actor.h

```diff
@@ -301,6 +303,7 @@ class ConflictingKeysImpl : public SpecialKeyRangeReadImpl {
 	Future<RangeResult> getRange(ReadYourWritesTransaction* ryw,
 	                             KeyRangeRef kr,
 	                             GetRangeLimits limitsHint) const override;
+	bool supportsTenants() const override { return true; };
```

Comment:
Have you done any evaluation of whether these modules do the right thing when run on a tenant transaction?

Reply:
I added a test case that is expected to not throw an error